python/taichi/ui/scene.py

```diff
@@ -125,10 +127,13 @@ def particles(self,
         vbo = get_vbo_field(centers)
         copy_vertices_to_vbo(vbo, centers)
         has_per_vertex_color = per_vertex_color is not None
+        has_per_vertex_radius = per_vertex_radius is not None
         if has_per_vertex_color:
             copy_colors_to_vbo(vbo, per_vertex_color)
+        if has_per_vertex_radius:
+            copy_texcoords_to_vbo(vbo, per_vertex_radius)
         vbo_info = get_field_info(vbo)
-        super().particles(vbo_info, has_per_vertex_color, color, radius)
+        super().particles(vbo_info, has_per_vertex_color, has_per_vertex_radius, color, radius)
```

Comment:
You haven't updated the `particles` function on the C++ side. Please have a look at 
https://github.com/taichi-dev/taichi/blob/master/taichi/python/export_ggui.cpp